frontend/src/components/me/config/Config.jsx

```diff
@@ -0,0 +1,296 @@
+import React from "react";
+import { MdCancel } from "react-icons/md";
+import {
+  BsFillCheckSquareFill,
+  BsFillPencilFill,
+  BsFillPlusCircleFill,
+  BsFillTrashFill,
+} from "react-icons/bs";
+import { Button, Col, Container, FormGroup, Input, Row } from "reactstrap";
+import useTitle from "react-use/lib/useTitle";
+import { usePluginConfigurationStore } from "../../../stores";
+import {
+  createCustomConfig,
+  CUSTOM_CONFIG_URI,
+  deleteCustomConfig,
+  updateCustomConfig,
+} from "./api";
+
+function isJSON(str) {
+  if (str === '""') return true;
+  try {
+    return Boolean(JSON.parse(str) && !!str);
+  } catch (e) {
+    return false;
+  }
+}
+
+export function isValidConfig(item) {
+  if (!item.type) {
+    addToast("Invalid config!", "Please select a type", "danger", true);
+    return false;
+  }
+  if (!item.plugin_name) {
+    addToast("Invalid config!", "Please select a plugin", "danger", true);
+    return false;
+  }
+  if (!item.attribute) {
+    addToast("Invalid config!", "Please select an attribute", "danger", true);
+    return false;
+  }
+  if (!isJSON(item.value)) {
+    addToast(
+      "Invalid config!",
+      "Please enter a JSON-compliant value",
+      "danger",
+      true
+    );
+    return false;
+  }
+  return true;
+}
+
+export function filterEmptyParams(plugins) {
+  const objectFilter = (obj, predicate) =>
+    Object.keys(obj)
+      .filter((key) => predicate(obj[key]))
+      .reduce((res, key) => Object.assign(res, { [key]: obj[key] }), {});
+  return objectFilter(
+    plugins,
+    (item) => item.params && Object.keys(item.params).length > 0
+  );
+}
+
+export default function Config() {
+  console.debug("Config rendered!");
+
+  useTitle("IntelOwl | Config", {
+    restoreOnUnmount: true,
+  });
+
+  const [
+    analyzers,
+    connectors,
+    retrieveAnalyzersConfiguration,
+    retrieveConnectorsConfiguration,
+  ] = usePluginConfigurationStore((state) => [
+    filterEmptyParams(state.analyzersJSON),
+    filterEmptyParams(state.connectorsJSON),
+    state.retrieveAnalyzersConfiguration,
+    state.retrieveConnectorsConfiguration,
+  ]);
+
+  const [respData, Loader, refetchConfig] = useAxiosComponentLoader(
+    {
+      url: CUSTOM_CONFIG_URI,
+    },
+    (resp) => resp.filter((item) => !item.organization)
+  );
+
+  const refetchAll = () => {
+    refetchConfig();
+    retrieveAnalyzersConfiguration();
+    retrieveConnectorsConfiguration();
+  };
+
+  return (
+    <Container>
+      <h4>Your custom configuration</h4>
+      <Loader
+        render={() => (
+          <Formik initialValues={{ config: respData }} onSubmit={null}>
+            {({ values, setFieldValue }) => (
+              <Form>
+                <FieldArray name="config">
+                  {({ remove, push }) => (
+                    <FormGroup row>
+                      <Col>
+                        {values.config && values.config.length > 0
+                          ? values.config.map((item, index) => {
```

Comment:
same logic of the comment above, please change "item" with "plugin"

Reply:
done